JK-548: add abbreviated alias for commands (#188)
jikkenio/jikken: CHANGELOG.md

- OpenApi test generation can now produce variable definitions for generating test data
- Stages in multi-stage tests can now have names via the `name` field
- New `NOW` and `NOW_UTC` built-in global variables, which contain the current timestamp in local and UTC time

> - New `FORMAT` command which auto formats your test files
> - New `VALIDATE` command for basic test file validation and it provides a mechanism to generate PlatformIds

Reviewer: ```suggestion
- New `format` command which auto-formats your test files
- New `validate` command for basic test file validation. which provides a mechanism to generate Platform IDs
```
